### DISCUSSION TITLE
Release notes?

### DISCUSSION THREAD

**Discussion by kds112** [devemux86/cruiser#249] [Q&A] [answered]
First I am very happy that Cruiser is developing very fast and each update contains interesting new features. But I don't have the time to follow all the discussions in Github and therefore it is a little bit surprising to detect  some of the new features after an update is released... 
Is it possible to add some kind of release notes in the setting "About Cruiser"? Just a short list of bullet points what is new since the last update? Maybe it could be a link to a website (in this Github repository?)?

**Comment by devemux86** [comment]
@kds112 thanks for the suggestion. Cruiser development is at full speed!

I could create a pinned topic with each release to mention the new features?

Note that the release notes are always mentioned on [Google Play](https://play.google.com/store/apps/details?id=gr.talent.cruiser) updates:

![Cruiser](https://github.com/devemux86/cruiser/assets/3484020/50dbaa01-cc30-476d-a546-39ff29538f96)

**Comment by devemux86** [comment, marked as answer]
Please see the pinned announcement topic:

- https://github.com/devemux86/cruiser/discussions/251

If anyone sees that we need improvements on informing users about new versions and features of Cruiser,
please let me know! :slightly_smiling_face:

**Reply by kds112** [reply]
Thanks a lot! I would suggest a permanent topic "What's new?" for all upcoming updates. Then you can check the features not only of the latest update....
And you could add a link to this topic in the setting "About Cruiser..." :-)
